src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -493,16 +493,12 @@ private void dominantSpeakerChanged()
     {
         Endpoint dominantSpeaker = speechActivity.getDominantEndpoint();
 
-        if (logger.isTraceEnabled())
-        {
-            logger.trace(
-                    "The dominant speaker in conference " + getID()
+        logger.info("The dominant speaker in conference " + getID()
```

Comment:
We could potentially keep the if statement to guard against unnecessary string concatenation.